### PR TITLE
Add useful vs-code settings for standarisation.

### PR DESCRIPTION
**Pull request by GuptaSiddhant**
I added a couple of useful extensions and many vs-code setttings that will help produc better and standarised code.

### CODE REVIEW

**Review by bjornallvin — CHANGES_REQUESTED** [review]
Hi @GuptaSiddhant. Thanks for the PR.  I like it.  Although I dont think we should mandate how VSCode should look for everyone.  Theme, colors and icons are personal choices I think, and it will not have any effect of the code produced.  So we need to remove some of the changes I think.

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,55 @@
 {
+  // Editor
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "editor.bracketPairColorization.enabled": true,
+  "editor.guides.bracketPairsHorizontal": "active",
+  "editor.fontFamily": "Fira Code, Menlo, Monaco, 'Courier New', monospace",
+  "editor.fontLigatures": true,
```
Comment (bjornallvin) [inline .vscode/settings.json:8]: I would remove these

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,4 +1,55 @@
...
+  // Workbench
+  "workbench.activityBar.visible": true,
+  "workbench.colorTheme": "Visual Studio Dark",
+  "workbench.colorCustomizations": {
+    "activityBar.background": "#000000",
+    "activityBar.activeBorder": "#A100FF" // Accenture Purple
+  },
+  "workbench.iconTheme": "material-icon-theme",
+  "material-icon-theme.folders.theme": "specific",
+  "material-icon-theme.activeIconPack": "react_redux",
```
Comment (bjornallvin) [inline .vscode/settings.json:25]: I would remove these

**Comment by bjornallvin** [comment]
@GuptaSiddhant whats the .devcontainer folder for? Seems to be some local configuration for Docker?

File: .vscode/extensions.json [hunk 3]
```diff
@@ -3,10 +3,11 @@
     "dsznajder.es7-react-js-snippets",
     "dbaeumer.vscode-eslint",
     "formulahendry.auto-rename-tag",
-    "coenraads.bracket-pair-colorizer",
```
Comment (GuptaSiddhant) [inline .vscode/extensions.json:6]: Not needed anymore with VS Code's native (and better) bracket pairing.

**Comment by GuptaSiddhant** [comment]
> @GuptaSiddhant whats the .devcontainer folder for? Seems to be some local configuration for Docker?

`.devcontainer` allows you to setup a standard docker container for [Github Codespaces](https://github.com/features/codespaces). 

I just added some basic NodeJS setup. We can extend it to do some amazing things like setup whole VS code env on the go and develop using GitHub's servers.

I can remove the config files but keeping them won't affect local development in any way.

**Comment by bjornallvin** [comment]
I think we should leave those out of the repo until we decide to actually use Codespaces for development fro everyone. To not create confusion.

**Review by bjornallvin — APPROVED** [review]

**Pull request merged**